frontend/src/routes/[[lang=locale]]/candidate/(protected)/questions/+page.svelte

```diff
@@ -5,103 +5,88 @@
   import {t} from '$lib/i18n';
   import {getRoute, Route} from '$lib/utils/navigation';
   import {getContext} from 'svelte';
-  import {get} from 'svelte/store';
   import {Warning} from '$lib/components/warning';
   import {
     QuestionsStartPage,
     AnsweredQuestion,
     UnAnsweredQuestion
   } from '$lib/candidate/components/questionsPage';
-  import type {Question} from '$lib/types/candidateAttributes';
   import type {CandidateContext} from '$lib/utils/candidateStore';
 
   const {
-    basicInfoFilledStore,
-    opinionQuestionsFilledStore,
-    nofUnansweredOpinionQuestionsStore,
-    questionsStore,
-    answersStore,
-    progressStore,
-    questionsLockedStore
+    opinionQuestions,
+    opinionAnswers,
+    progress,
+    questionsLocked,
+    unansweredRequiredInfoQuestions,
+    unansweredOpinionQuestions
   } = getContext<CandidateContext>('candidate');
 
-  $: questionsLocked = $questionsLockedStore;
-
-  let questions = get(questionsStore) ?? [];
-
-  let opinionQuestionsLeft: number | undefined;
-  nofUnansweredOpinionQuestionsStore?.subscribe((value) => {
-    opinionQuestionsLeft = value;
-  });
-
-  let opinionQuestionsFilled: boolean | undefined;
-  opinionQuestionsFilledStore?.subscribe((value) => {
-    opinionQuestionsFilled = value;
-  });
-
-  let basicInfoFilled: boolean | undefined;
-  basicInfoFilledStore?.subscribe((value) => {
-    basicInfoFilled = value;
-  });
-
-  $: answers = $answersStore;
-
-  let questionsByCategory: Record<string, Array<Question>>;
+  let questionsByCategory: Record<string, Array<QuestionProps>> | undefined;
 
   let loading = true;
   let unansweredCategories: Array<string> | undefined;
```

Comment:
I think it's best to initialise this as `let unansweredCategories = new Array<string>();`